src/controller/jobmonitor/monitor.go

```diff
@@ -49,28 +50,39 @@ type MonitorController interface {
 	ListWorkers(ctx context.Context, poolID string) ([]*jm.Worker, error)
 	// StopRunningJob stop the running job
 	StopRunningJob(ctx context.Context, jobID string) error
+	// ListQueue lists job queues
+	ListQueue(ctx context.Context) ([]*jm.Queue, error)
+	// StopPendingJob stop the pending job
+	StopPendingJob(ctx context.Context, jobType string) error
+	// PauseJobQueues suspend the all schedules or resume the all schedules
+	PauseJobQueues(ctx context.Context, jobType string, pause bool) error
```

Comment:
This function includes pause and resume operations, and use the parameter 'pause'  to choose, I think the function name is confusing.

Reply:
split to pauseJobQueue and resumeJobQueue